s3backgrounddelete/s3backgrounddelete/cortx_s3_util.py

```diff
@@ -102,13 +102,16 @@ def sign_request_v4(self, method=None, canonical_uri='/', canonical_query_string
 
        algorithm = 'AWS4-HMAC-SHA256'
 
-       if self._config.get_s3recovery_flag():
-           access_key = self._config.get_s3_recovery_access_key()
-           secret_key = self._config.get_s3_recovery_secret_key()
-       else:
-           access_key = self._config.get_cortx_s3_access_key()
-           secret_key = self._config.get_cortx_s3_secret_key()
-
+#       if self._config.get_s3recovery_flag():
```

Comment:
remove this instead of commenting the section out.

Reply:
done